Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,20 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+
+// the whole number part of num
+const wholeNumberPart = Math.floor(num);
+
+// the decimal part of num
+const decimalPart = num - wholeNumberPart
```

Comment:
Result is not what is expected. Value contains too many decimal places.

Reply:
Thanks a ton for the feedback! Just to clarify, when you mentioned "result is not what was expected," did you mean only the number of decimal places? If that’s the case, I looked into it and learned about .toFixed(4), which keeps things cleaner by limiting the decimal places.

I also learned that .toFixed() has a small catch—it actually returns the result as a string rather than a number. So, to make sure I get a number again, I’m using the + sign before the expression. It’s a quick way to convert it back to a number without affecting the value. I will go ahead and make the required changes. Thanks again for your time.